### PR TITLE
Added normalize column to export grade book to remote gradebook

### PR DESCRIPTION
**Pull request by amir-qayyum-khan**
### Background
This PR resolves https://github.com/mitocw/edx-platform/issues/147, resolves  https://github.com/mitocw/edx-platform/issues/187 and resolves https://github.com/mitocw/edx-platform/issues/188

### What is done in this PR
**Studio Updates:** None
**LMS Updates:** 
- Added checkbox on legacy dashboard to indicate grades are normalize or not
- On grade book export added column normalize with value 1 if gadebook is normalize else 0.

@pdpinch @pwilkins 

- screen shot 2016-02-10 at 5 58 49 pm

- screen shot 2016-02-10 at 6 00 57 pm

### CODE REVIEW

**Comment by pdpinch** [comment]
I think you may also need to cherry-pick https://github.com/edx/edx-platform/pull/10213 and https://github.com/edx/edx-platform/pull/10395 as well. These precede @pwilkins's commit.

**Comment by pdpinch** [comment]
Unfortunately, I'm having trouble getting this to run. I think you may have a dependency on a post-cypress version of grades.py

See the stacktrace at http://dpaste.com/07YXX1W

I tried just cherry-picking the commit that contained the missing function, but it's huge and came with a bunch of other dependencies. I'm not sure if it's better to refactor to avoid using `_calculate_score_for_modules()` or to backport it to our branch of cypress.

**Comment by amir-qayyum-khan** [comment]
@pdpinch done with testing on cypress. Also I have to cherry pick some comments.

File: AUTHORS [hunk 1]
```diff
@@ -225,4 +225,36 @@ Alessandro Verdura <finalmente2@tin.it>
 Sven Marnach <sven@marnach.net>
 Richard Moch <richard.moch@gmail.com>
 Albert Liang <albertliangcode@gmail.com>
+<<<<<<< HEAD
```
Comment (noisecapella) [inline AUTHORS]: Where did this come from?

**Comment by amir-qayyum-khan** [comment]
@noisecapella Fix issues 

sorry I dont understand:
>> This doesn't fix the case where len(student.grades) == aidx.
Can you refer me the fix if it is somewhere in cypress or dogwood PRs or can you refer me to the issue for more detail?

cc @pdpinch

**Comment by noisecapella** [comment]
Can you make a separate PR with the cherry picked commits? The cherry picked commits also need to be cleaned up. `[LTI Provider] Grade passback for non-leaf blocks.` appears three times, and `Added max score to output of get_student_grade_summary_data.` includes a lot of stuff that wasn't part of his original commit

**Comment by amir-qayyum-khan** [comment]
Done @noisecapella 
https://github.com/mitocw/edx-platform/pull/209

**Pull request closed without merging**